lib/src/main/kotlin/starknet/crypto/NativeLoader.kt

```diff
@@ -0,0 +1,33 @@
+package starknet.crypto
+
+import java.io.File
```

Comment:
I'd prefer we not use io.File at all. This is an old api. `nio.file` is newer and provides a better apis.